cinema/views.py

```diff
@@ -127,6 +126,54 @@
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "actors",
+                type=str,
+                description="Filter by actors id",
+                required=False,
+            ),
+            OpenApiParameter(
+                "actors",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by actors id",
+            ),
+        ]
+    )
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "genres",
+                type=str,
+                description="Filter by genres id",
+                required=False,
+            ),
+            OpenApiParameter(
+                "genres",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by genres id",
+            ),
+        ]
+    )
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "title",
+                type=str,
+                description="Filter by title id",
+                required=False,
+            ),
+            OpenApiParameter(
+                "title",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by title id",
+            ),
+        ]
+    )
+    def list(self, request, *args, **kwargs):
```

Comment:
This is a code style issue. The `list` method is not doing anything beyond calling the superclass method. If you're not adding any additional logic, you can just omit this method.